.travis.yml

```diff
@@ -7,14 +7,18 @@ services:
 
 arch:
 - amd64
-# - ppc64le # temporary disable until we have it under control again
+- ppc64le
 
 os: linux
 
 install:
 - git reset --hard
 
 script:
+- |
+  if [[ $TRAVIS_CPU_ARCH == "ppc64le" ]]; then
```

Comment:
I tried to use this too, but that does not help. When playing with it, it turned out that every `-` is executed in its own subshell and all backgroupd processes are terminated. This means that we start the background process and it quits immediately.

Reply:
From what I've been testing it did the trick for me, I used to see the issue of the no output pretty often and with this I didn't see it once in a lot of runs.

For example in the current run on the CI: https://travis-ci.org/github/kubevirt/kubevirt/jobs/672952277
You can see the keep alive message in Lines 1133 and 1213. The keep alive process only gets killed at the end when I use the kill command to its PID, as seen in line 2737:
```
1827 Terminated              while sleep 9m; do
    echo "Long running job - keep alive";
done
```

I think I found the run in which you ran into an issue when trying a similar approach: https://travis-ci.org/github/kubevirt/kubevirt/jobs/665757942#L99
I was taking a look at it, and what I found interesting is if you look at the line 1072 it seems like the output got cropped mid print: ” 	versi”
I think you somehow got unlucky and an actual stall seemed to have happened. I’ve been attempting to recreate it with your patch, but the process keeps running in background and it actually prints the keep alive: https://travis-ci.com/github/murilofv/kubevirt/builds/160393191
The process only gets killed at the end of the build

Reply:
I am convinced now. Thanks for the detailed analysis.